dali-mqtt-daemon.py

```diff
@@ -296,6 +297,14 @@ def create_mqtt_client(
             from dali.driver.hasseb import SyncHassebDALIUSBDriver
 
             dali_driver = SyncHassebDALIUSBDriver()
+            firmware_version = float(dali_driver.readFirmwareVersion())
+            if firmware_version < MIN_HASSEB_FIRMWARE_VERSION:
+                logger.error("Using a script requires newest hasseb firmware")
```

Comment:
"a script" ? should be dali2mqtt

Reply:
I changed it

Reply:
I think this should mapped to script name via os.path.basename(__file__) or something similar